cms/standard_pages/models.py

```diff
@@ -7,9 +7,10 @@
 from cms.core.blocks.stream_blocks import CoreStoryBlock
 from cms.core.fields import StreamField
 from cms.core.models import BasePage
+from cms.private_media.models import MediaParentMixin
 
 
-class InformationPage(BasePage):  # type: ignore[django-manager-missing]
+class InformationPage(MediaParentMixin, BasePage):  # type: ignore[django-manager-missing]
```

Comment:
@MebinAbraham NOTE: This mixin must be used any model class that we want to be seen as a 'media parent'. That might also include snippets in future.

Reply:
suggestion: given the steer was to always treat assets as private until the pages they are attached to are published, why not add it to `BasePage`?